spec/teaspoon/configuration_spec.rb

```diff
@@ -149,6 +152,19 @@
 
   end
 
+  describe ".coverage_ignored_files" do
+
+    it "returns an empty list if nothing was set" do
+      expect(subject.coverage_ignored_files).to eq([])
+    end
+
+    it "returns an array of  if they were comma separated" do
```

Comment:
ok, so when a suite is rendered via the controller, we usually tell it which files to include, and at that time we can determine if we want to instrument it or not (by adding an "instrument=true" param to the asset). Then when the asset is loaded the instrumenter checks for that param and does or does not instrument the file (the instrumenter hooks into sprockets).

With requirejs, when those assets are requested, requirejs doesn't know which ones should have the param, and the instrumenter doesn't know either because it doesn't know what suite was rendered.